website/docs/r/lb_server_pool.html.markdown

```diff
@@ -0,0 +1,129 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_lb_server_pool"
+sidebar_current: "docs-vcd-resource-lb-server-pool"
+description: |-
+  Provides a NSX load balancer server pool data source.
+---
+
+# vcd\_lb\_server\_pool
+
+Provides a vCloud Director Edge Gateway Load Balancer Server Pool data source. A Server Pool defines
```

Comment:
"A Server Pool defines" -> perhaps "A Server Pool establishes" or "creates"?
Just to avoid the clash with the "defined" in the next statement.

Reply:
"Refactored" the paragraph. Hope it is more readable now.